src/controller/artifact/processor/chart/chart.go

```diff
@@ -16,17 +16,11 @@ package chart
 
 import (
 	"context"
-	"encoding/json"
```

Comment:
Please keep this file unchanged, this code is used to handle OCI chart artifact.

Reply:
fixed.